ideascube/serveradmin/catalog.py

```diff
@@ -18,6 +18,14 @@
 
 from .systemd import Manager as SystemManager, NoSuchUnit
 
+# We do not use functool.partial cause we want to mock stderr for unittest
+# If we use partial we keep a ref to the original sys.stderr and output is not
+# captured.
+def printerr(*args, **kwords):
```

Comment:
Also I think printerr could be on a util.py module (even in the root one), so we can reuse it.